docs/policy.json.md

```diff
@@ -146,13 +146,25 @@ This requirement requires an image to be signed with an expected identity, or ac
     "keyType": "GPGKeys", /* The only currently supported value */
     "keyPath": "/path/to/local/keyring/file",
     "keyData": "base64-encoded-keyring-data",
-    "signedIdentity": identity_requirement
+    "signedIdentity": identity_requirement,
+    "referencesByDigest": refs_by_digest_requirement
 }
 ```
 <!-- Later: other keyType values -->
 
 Exactly one of `keyPath` and `keyData` must be present, containing a GPG keyring of one or more public keys.  Only signatures made by these keys are accepted.
 
+The `referencesByDigest` field controls whether an reference with a digest should be taken into consideration when checking signatures or be rejected altogheter.
+One of the following alternatives are supported:
+
+- References by digest are allowed (default). In this case, the digest should match the one from the signature for the verification to be successful.
+  ```json
+  {"referencesByDigest: allow"}
```

Comment:
The `{…}` form for `signedIdentity` is used because a `signedIdentity` value _is_ such an object; but the above is not a valid `"type":"signedBy"` object. Perhaps, just use a bulleted list like `- "allow": Reference by digest are …`